### PR TITLE
Raylife AP Automated Test - LPS-160911 | master

### PR DESCRIPTION
**Pull request by itstali**
https://issues.liferay.com/browse/LPS-160911

### CODE REVIEW

**Comment by itstali** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:site-initializer-raylife-ap:nocompile

File: modules/apps/site-initializer/site-initializer-raylife-ap/site-initializer-raylife-ap-test/src/testFunctional/paths/RaylifeApplicationModal.path [hunk 1]
```diff
@@ -11,6 +11,11 @@
 
 <tbody>
 
+<tr>
+	<td>ANY_DRIVER_BUTTON</td>
+	<td>xpath=(//div[contains(@class, 'bg-neutral-0')]/descendant::button)[${key_index}]</td>
+	<td></td>
+</tr>
```
Comment (luanakarla) [inline modules/apps/site-initializer/site-initializer-raylife-ap/site-initializer-raylife-ap-test/src/testFunctional/paths/RaylifeApplicationModal.path:18]: Reminder: when the devs refactor the code, don't forget to redo this path.

**Review by luanakarla — APPROVED** [review]
Hey @itstali, thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by luanakarla** [comment]
ci:forward

**Pull request closed without merging**